components/eam/src/chemistry/mozart/mo_exp_sol.F90

```diff
@@ -109,6 +113,65 @@ subroutine exp_sol( base_sol, reaction_rates, het_rates, extfrc, delt, xhnm, nco
                 base_sol(i,k,l) = base_sol(i,k,l)*exp(-delt*loss(i,k,m)/base_sol(i,k,l)) + delt*(prod(i,k,m)+ind_prd(i,k,m))
              end do
           end do
+#if (defined MODAL_AERO_5MODE)       
+       elseif ( trim(solsym(l)) == 'SO2') then 
+         do i = 1,ncol
+             do k = 1,pver
+                chem_prod(i,k,l) = prod(i,k,m)+ind_prd(i,k,m)
+                chem_loss(i,k,l) = - 1.0_r8*loss(i,k,m)
+                base_sol(i,k,l) = base_sol(i,k,l) + delt * (prod(i,k,m) + ind_prd(i,k,m) - loss(i,k,m))
+               !base_sol(i,k,l) = base_sol(i,k,l)*exp(-delt*loss(i,k,m)/base_sol(i,k,l)) + delt*(prod(i,k,m)+ind_prd(i,k,m))
+             end do
+         end do
+       elseif ( trim(solsym(l)) == 'H2SO4') then 
+         do i = 1,ncol
+             do k = 1,pver
+                chem_prod(i,k,l) = prod(i,k,m)+ind_prd(i,k,m)
+                chem_loss(i,k,l) = - 1.0_r8*loss(i,k,m)
+                base_sol(i,k,l) = base_sol(i,k,l) + delt * (prod(i,k,m) + ind_prd(i,k,m) - loss(i,k,m))
+                !base_sol(i,k,l) = base_sol(i,k,l)*exp(-delt*loss(i,k,m)/base_sol(i,k,l)) + delt*(prod(i,k,m)+ind_prd(i,k,m))
+             end do
+         end do
+      elseif ( trim(solsym(l)) == 'DMS') then 
```

Comment:
Only SO2 and H2SO4 need to use a different equation. DMS can use the same equation as other gases. I suggest removing DMS from the changes here and other relevant places.